build broken after #194

After 9c70ac9, the generated `lib/Makefile` ends up with:

```Makefile
AR=     ar cr ${LIB} ${OBJ}
RANLIB= ranlib ${LIB}
# ...
${LIB}: ${OBJ}
        ${AR} cr ${LIB} ${OBJ}
        ${RANLIB} ${LIB}
```

So the build fails with an error like:
```
ar cr liblsof.a ckkv.o cvfs.o dvch.o fino.o isfn.o lkud.o pdvn.o prfp.o ptti.o rdev.o regex.o rmnt.o rnam.o rnch.o rnmh.o snpf.o cr liblsof.a ckkv.o cvfs.o dvch.o fino.o isfn.o lkud.o pdvn.o prfp.o ptti.o rdev.o regex.o rmnt.o rnam.o rnch.o rnmh.o snpf.o
ar: cr: No such file or directory
make[1]: *** [liblsof.a] Error 1
make: *** [lib/liblsof.a] Error 2
```

since the `cr ${LIB} ${OBJ}` arguments end up duplicated.

--- a/00DIST
+++ b/00DIST
@@ -5298,8 +5298,6 @@ July 14, 2018
 		Fix broken LSOF_CFLAGS_OVERRIDE.
 		Provided by Fabrice Fontaine in #172.
 
-		Fix broken AR and RANLIB when cross-compiling.
-		Provided by Fabrice Fontaine in #194.
 
 		[linux] Remove sysvlegacy function.
 		Provided by Fabrice Fontaine in #195.

--- a/lib/Makefile.skel
+++ b/lib/Makefile.skel
@@ -21,8 +21,8 @@ OBJ=	ckkv.o cvfs.o dvch.o fino.o isfn.o lkud.o pdvn.o prfp.o \
 all:	${LIB}
 
 ${LIB}:	${OBJ}
-	${AR} cr ${LIB} ${OBJ}
-	${RANLIB} ${LIB}
+	${AR}
+	${RANLIB}
 
 clean:	FRC
 	rm -f ${LIB} ${OBJ} errs Makefile.bak a.out core

